javascript/builtins/Error.json

```diff
@@ -105,6 +105,57 @@
             }
           }
         },
+        "cause": {
```

Comment:
@apple-web-evangelist sorry, I missed this previously. This nesting is off by one level. It's at `javascript.builtins.Error.options_cause_parameter` instead of `javascript.builtins.Error.Error.options_cause_parameter`.

(that is, it's a sibling of the constructor data, rather than a descendant of it)